lib/libc/common/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@
 
 zephyr_library()
 zephyr_library_sources_ifdef(CONFIG_COMMON_LIBC_TIME source/time/time.c)
+zephyr_library_property(ALLOW_EMPTY TRUE)
```

Comment:
This is already being addressed as part of https://github.com/zephyrproject-rtos/zephyr/pull/56309.

Reply:
Thank you, I have missed that such a change is pending.